Annotations/Crafttweaker_Annotations/src/main/java/com/blamejared/crafttweaker_annotations/annotations/TaggableElement.java

```diff
@@ -0,0 +1,23 @@
+package com.blamejared.crafttweaker_annotations.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface TaggableElement {
+    
+    /**
+     * The name of the registry that holds this element.
+     *
+     * Examples:
+     * - "minecraft:item" for Item
+     * - "minecraft:worldgen/biome" for Biome
+     *
+     * @return The name of the registry that holds this element.
+     */
+    String value();
```

Comment:
We're currently not using any intelliJ Annotations right now, and with the current setup the annotations won't be included in the maven pom file, so I don't know if it would really benefit anyone.